core/domain/collection_services.py

```diff
@@ -479,13 +480,18 @@ def get_collection_ids_matching_query(query_string, cursor=None):
 
     Args:
         query_string: str. The search query string.
+        cursor: Cursor or None. Cursor pointing to the start of the collection
```

Comment:
I didn't know we had a special cursor object. What do you get when you do `print type(cursor)`? Is it just a str?

Also, I think the explanation should be more like: "Cursor indicating where, in the list of collections, to start the search from.", since the "start of the collection" doesn't really make sense.